tools/charmm36/charmm36.py

```diff
@@ -0,0 +1,201 @@
+
```

Comment:
Added tools must be properly described in the manual and have a README file in their folder explaining its specific use and a one line description added to tools/README.

This one doesn't look like it is ready for general use but rather specific for your personal use. I assume its commit is accidental and thus it should be removed.

Reply:
done